test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -18,6 +18,13 @@ def test_ort_perf_tuning_pass(config, tmp_path):
     p = create_pass_from_dict(OrtPerfTuning, config, disable_search=True)
     output_folder = str(tmp_path / "onnx")
 
+    # check config
+    # pylint: disable=protected-access
+    assert p._config["device"] is not None
```

Comment:
do we need add test cases that user explicitly specify provider_list?

Reply:
Updated. Add will check it in line 34.